integration/nwo/orion/platform.go

```diff
@@ -220,164 +185,29 @@ func (p *Platform) Members() []grouper.Member {
-				Expect(err).NotTo(HaveOccurred())
-				break
-			} else {
-				logger.Infof("cleanup container [%s], skipped", name)
-			}
-		}
-	}
 
-	volumes, err := p.DockerClient.ListVolumes(docker.ListVolumesOptions{})
+	err = d.CreateNetwork(p.NetworkID)
 	Expect(err).NotTo(HaveOccurred())
-	for _, i := range volumes {
-		if strings.HasPrefix(i.Name, p.NetworkID) {
-			logger.Infof("cleanup volume [%s]", i.Name)
-			err := p.DockerClient.RemoveVolumeWithOptions(docker.RemoveVolumeOptions{
-				Name:  i.Name,
-				Force: false,
-			})
-			Expect(err).NotTo(HaveOccurred())
-			break
-		}
-	}
 
-	images, err := p.DockerClient.ListImages(docker.ListImagesOptions{All: true})
-	Expect(err).NotTo(HaveOccurred())
-	for _, i := range images {
-		for _, tag := range i.RepoTags {
-			if strings.HasPrefix(tag, p.NetworkID) {
-				logger.Infof("cleanup image [%s]", tag)
-				err := p.DockerClient.RemoveImage(i.ID)
-				Expect(err).NotTo(HaveOccurred())
-				break
-			}
-		}
-	}
-
-	err = p.DockerClient.RemoveNetwork(nw.ID)
-	Expect(err).NotTo(HaveOccurred())
+	p.RunOrionServer()
+	p.InitOrionServer()
 }
 
-func (p *Platform) RunOrionServer() {
-	ctx := context.Background()
-	cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
-	if err != nil {
-		panic(err)
-	}
-
-	net, err := p.DockerClient.NetworkInfo(p.NetworkID)
-	if err != nil {
-		panic(err)
-	}
-
-	strNodePort := strconv.Itoa(int(p.nodePort))
-	strPeerPort := strconv.Itoa(int(p.peerPort))
-
-	containerName := fmt.Sprintf("%s.orion.%s", p.NetworkID, p.Topology.TopologyName)
-	resp, err := cli.ContainerCreate(ctx, &container.Config{
-		Hostname: containerName,
-		Image:    ServerImage,
-		Tty:      false,
-		ExposedPorts: nat.PortSet{
-			nat.Port(strNodePort + "/tcp"): struct{}{},
-			nat.Port(strPeerPort + "/tcp"): struct{}{},
-		},
-	}, &container.HostConfig{
-		Mounts: []mount.Mount{
-			{
-				Type: mount.TypeBind,
-				// Absolute path to
-				Source: p.ledgerDir(),
-				Target: "/etc/orion-server/ledger",
-			},
-			{
-				Type: mount.TypeBind,
-				// Absolute path to
-				Source: p.cryptoDir(),
-				Target: "/etc/orion-server/crypto",
-			},
-			{
-				Type: mount.TypeBind,
-				// Absolute path to
-				Source: p.configDir(),
-				Target: "/etc/orion-server/config",
-			},
-		},
-		PortBindings: nat.PortMap{
-			nat.Port(strNodePort + "/tcp"): []nat.PortBinding{
-				{
-					HostIP:   "0.0.0.0",
-					HostPort: strNodePort,
-				},
-			},
-			nat.Port(strPeerPort + "/tcp"): []nat.PortBinding{
-				{
-					HostIP:   "0.0.0.0",
-					HostPort: strPeerPort,
-				},
-			},
-		},
-	}, &network.NetworkingConfig{
-		EndpointsConfig: map[string]*network.EndpointSettings{
-			p.NetworkID: {
-				NetworkID: net.ID,
-			},
-		},
-	}, nil, containerName)
-	if err != nil {
-		panic(err)
-	}
+func (p *Platform) Cleanup() {
+	dockerClient, err := docker.GetInstance()
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
can we take the chance and add an error message to the new Expect introduced?
In this case, something like
Expect(err).NotTo(HaveOccurred(), "failed to get docker client")

Reply:
I added more context to the actual error.